src/axom/spin/BVH.hpp

```diff
@@ -365,6 +375,7 @@ bool InRight( const internal::linear_bvh::Vec< FloatType, 2>& point,
 template< int NDIMS, typename ExecSpace, typename FloatType >
 BVH< NDIMS, ExecSpace, FloatType >::BVH( const FloatType* boxes,
                                          IndexType numItems ) :
+  m_scaleFactor( 1.001 ),
```

Comment:
Do we want a magic number here?

Reply:
It is setting a default scale factor -- It will be better to have a `constexpr` variable to set the scale factor and then use the `constexpr` variable here. I'll make this change.